src/Nordea/Components/InfoLabel.elm

```diff
@@ -0,0 +1,56 @@
+module Nordea.Components.InfoLabel exposing (view, viewWarning)
+
+import Css
+    exposing
+        ( backgroundColor
+        , borderRadius
+        , displayFlex
+        , fitContent
+        , flexDirection
+        , height
+        , marginBottom
+        , marginRight
+        , maxWidth
+        , padding
+        , rem
+        , row
+        , width
+        )
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.Text as Text
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.Icons as Icon
+import Nordea.Themes as Themes
+
+
+view : List (Attribute msg) -> List (Html msg) -> Html msg
+view attrs children =
+    Html.div
+        (infoLabelContainerStyle :: css [ Themes.backgroundColor Themes.SecondaryColor Colors.blueCloud ] :: attrs)
+        [ Icon.info [ css [ width (rem 1.5), height (rem 1.5), marginRight (rem 0.5) ] ]
+        , Text.bodyTextSmall
+            |> Text.view [] children
+        ]
+
+
+viewWarning : List (Attribute msg) -> List (Html msg) -> Html msg
+viewWarning attrs children =
+    Html.div
+        (infoLabelContainerStyle :: css [ backgroundColor Colors.yellow ] :: attrs)
+        [ Icon.info [ css [ width (rem 1.5), height (rem 1.5), marginRight (rem 0.5) ] ]
+        , Text.bodyTextSmall
+            |> Text.view [] children
+        ]
```

Comment:
Unsure how to make the use of this optimal. `InfoLabel.viewWarning [] []` doesn't feel optimal, but at the same time I would like to avoid the `Component.componentType |> Component.view [] []` pattern since it feels so verbose. Any thoughts?